cvat/apps/engine/serializers.py

```diff
@@ -183,7 +183,8 @@ class Meta:
         write_once_fields = ('overlap', 'segment_size', 'image_quality')
         ordering = ['-id']
 
-    def create(self, validated_data):
+    @staticmethod
+    def create(validated_data):
```

Comment:
The same, it is an overloaded method of ModelSerializer. I don't think that we should remove self here.